includes/class-sensei-settings-api.php

```diff
@@ -211,16 +211,21 @@
 					'href'  => '#' . esc_attr( $k ),
 					'name'  => esc_attr( $v['name'] ),
 					'class' => esc_attr( $classes ),
+					'badge' => esc_html( $v['badge'] ),
```

Comment:
This needs a null check as it's not added for all the tabs:
```
( ! ) Notice: Undefined index: badge in /Users/yscik/dev/sensei/includes/class-sensei-settings-api.php on line 214
```

Reply:
Fixed! a8d586346e5f670b29d6cb7b4911fb5a085783ad